index.html

```diff
@@ -5,25 +5,35 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" type="text/css" href="style.css">
+        <link rel="stylesheet" type="text/css" href ="desktop.css">
         <title>Portfolio Setup Mobile First</title>
     </head>
     <body>
         <!--Header section-->
         <header class="header">
-            <a id="logo" href="#">Allan K. </a>
-            <nav>
+            <nav class="mobile-menu">
+                <a class="logo" href="#">Allan K. </a>
                 <a href="#"><img class="menuimg" src="assets/header/menu.png" alt="menu"/></a>
             </nav>
+            <nav class="menu-desktop">
+                <a class="logo" href="#">Allan K. </a>
+                <ul class="nav-links">
+                    <li class="nav-link"><a href="#portfolio">Portfolio</a></li>
+                    <li class="nav-link"><a href="#about">About</a></li>
+                    <li class="nav-link"><a href="#feedback">Contact</a></li>
+                    <li><a href="#"><img src="assets/header/Icon - Mail.svg" alt="mail"></a></li>
+                </ul>
+            </nav>           
         </header>
         <main class="content">
             <section class="welcome">
-                <article>
-                    <h1 class="main-title">Hey There. I'm Allan.</h1>
+                <article class="hello">
+                    <h1 class="main-title"><span>Hey There. I'm Allan.</span></h1>span></h1>
```

Comment:
- [ ] As [suggested](https://github.com/allenkays/Portfolio-setup-and-mobile-first/pull/4#discussion_r1264563862), kindly make sure to fix the `span` tag. Right now, I can see that `span` tag is rendering as 👇
 
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/106119105/fbacc10a-ecd6-41d6-8477-e12d82f0d6f9)